core/src/xtdb/tx.clj

```diff
@@ -342,32 +346,26 @@
     (when-not (compare-and-set! !tx-state :open :committed)
       (throw (IllegalStateException. (str "Transaction marked as " (name @!tx-state)))))
 
-    (when fork-at
-      (throw (IllegalStateException. "Can't commit from fork.")))
-
     (fork/commit-doc-store-tx document-store-tx)
     (db/commit-index-tx index-store-tx)
 
     (log/debug "Transaction committed:" (pr-str tx))
 
     (let [{:keys [tx-events]} @!tx]
-      (bus/send bus (into {::xt/event-type ::indexed-tx,
-                           :submitted-tx tx,
-                           :committed? true
-                           ::txe/tx-events tx-events}
-                          (select-keys @!tx [:doc-ids :av-count :bytes-indexed])))))
+      (bus/send bus (merge {::xt/event-type ::indexed-tx,
```

Comment:
why the change from `into` to `merge` here? `into` uses transient collections under the hood, and no varargs, so is more performant than `merge` (which is essentially `reduce conj`)

Reply:
I've reverted back to `into`